internal/bots/telegram/handlers/summarize/smr_command.go

```diff
@@ -39,20 +31,16 @@ func (h *Handlers) Handle(c *tgbot.Context) (tgbot.Response, error) {
 		return nil, tgbot.NewExceptionError(err)
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*2)
-	defer cancel()
+	err = h.smrQueue.AddTask(types.TaskInfo{
+		Platform:  smr.FromPlatformTelegram,
+		Url:       urlString,
+		ChatID:    c.Update.Message.Chat.ID,
+		MessageID: processingMessage.MessageID,
+	})
 
-	summarization, err := h.smr.SummarizeInputURL(ctx, urlString, smr.FromPlatformTelegram)
 	if err != nil {
-		if errors.Is(err, smr.ErrContentNotSupported) {
-			return nil, tgbot.NewMessageError("暂时不支持量子速读这样的内容呢，可以换个别的链接试试。").WithEdit(&processingMessage)
-		}
-		if errors.Is(err, smr.ErrNetworkError) || errors.Is(err, smr.ErrRequestFailed) {
-			return nil, tgbot.NewMessageError("量子速读的链接读取失败了哦。可以再试试？").WithEdit(&processingMessage)
-		}
-
-		return nil, tgbot.NewMessageError("量子速读失败了。可以再试试？").WithEdit(&processingMessage)
+		return nil, tgbot.NewMessageError("量子速读请求发送失败了，可以再试试？").WithEdit(&processingMessage)
```

Comment:
```suggestion
		return nil, tgbot.NewExceptionError(err).WithMessage("量子速读失败了。可以再试试？").WithEdit(&processingMessage)
```

文案上不需要包含这种细节，可以用 ExceptionError + err 传递的方式处理掉